sieve/message.h

```diff
@@ -67,7 +67,7 @@ typedef enum {
     ACTION_UNMARK,
     ACTION_ENOTIFY,
     ACTION_NOTIFY,
-    ACTION_DENOTIFY
+    ACTION_DENOTIFY,
```

Comment:
Is this extra trailing comma deliberate?  Presumably it might annoy some compilers?

Reply:
This is perfectly valid, and excellent C style (C++ and perl too).  I do this all the time, deliberately, and have done so for years.  Future additions to this list will diff as added lines only, which is very readable.

Without the trailing comma, the next time someone adds to the list, the diff will have a noisy delete-and-two-adds, because you can't just add to the list, you also have to modify the old last one.

(JavaScript notably doesn't like this sort of thing; you might have tripped over that in the past.)